app/Services/TransactionSyncService.php

```
@@ -0,0 +1,177 @@
+     * @param Account $account
+     * @param int $maxDays
+     * @return array
+     */
+    public function calculateDateRange(Account $account, int $maxDays = 90, bool $forceMax = false): array
+    {
+        $dateTo = now()->format('Y-m-d');
+
+        //TODO validate
+
+        // If account has been synced before, sync from last sync date
+        if ($account->gocardless_last_synced_at && !$forceMax) {
```

PHPStan (2.1.15)

Access to an undefined property App\Models\Account::$gocardless_last_synced_at.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)